routes/constants.js

```diff
@@ -0,0 +1,7 @@
+function ACCOUNT_HAS_BEEN_MERGED () {
+  return 'ACCOUNT_HAS_BEEN_MERGED'
+}
+
+module.exports = {
+  ACCOUNT_HAS_BEEN_MERGED
+}
```

Comment:
Why is this a function rather than a JS object? ie 

`module.exports = { ACCOUNT_HAS_BEEN_MERGED: 'ACCOUNT_HAS_BEEN_MERGED' }`